File: 01.JS-Basic/More Exercises/PB - More Exercises/Simple Operations and Calculations/fishLand.js
function fishLand(input) {

    let palamudPrice = Number(input[0]) + Number(input[0]) * 0.6
    let safridPrice = Number(input[1]) + Number(input[1]) * 0.8
    let midiPrice = 7.50

    let palamudKg = Number(input[2])
    let safridKg = Number(input[3])
    let midiKg = Number(input[4])

    let palamudSpend = palamudPrice * palamudKg
    let safridSpend = safridKg * safridPrice
    let midiSpend = midiKg * midiPrice

    let total = palamudSpend + safridSpend + midiSpend

    console.log(total.toFixed(2))

}
fishLand(['6.9', '4.2', '1.5', '2.5', '1'])

File: JAVA-Basic/JavaBasic/src/com/company/FirstLesson/FirstStepsInCodingExercise/MoreExercises/Fishland.java
package com.company.FirstLesson.FirstStepsInCodingExercise.MoreExercises;

import java.util.Scanner;

public class Fishland {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        double scumriaPrice = Double.parseDouble(scan.nextLine());
        double cacaPrice = Double.parseDouble(scan.nextLine());
        double palamudKg = Double.parseDouble(scan.nextLine());
        double safridKg = Double.parseDouble(scan.nextLine());
        int midiKg = Integer.parseInt(scan.nextLine());

        double palamudPrice = scumriaPrice * 1.6;
        double safridPrice = cacaPrice * 1.8;
        double midiPrice = 7.5;

        double totalMoneyNeeded = palamudKg * palamudPrice + safridPrice * safridKg + midiPrice * midiKg;

        System.out.printf("%.2f", totalMoneyNeeded);
    }
}
